torch/csrc/jit/ir/ir.cpp

```diff
@@ -1595,6 +1595,21 @@ Node* Graph::createTupleSlice(Value* tup, int64_t beg, int64_t end) {
   return n;
 }
 
+Node* Graph::createEnumName(Value* e) {
```

Comment:
We don't actually need this custom create logic, bc there is only one possible schema for `prim::EnumName`, so you can use the normal schema driven insert logic: 

https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/ir/ir.h#L1181

Reply:
Agreed, but it is actually nice to have a method to create it, more discoverable and we can enforce type checking here.